internal/controller/operatorconfigmap_controller.go

```diff
@@ -410,6 +411,9 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI() error {
 		if c.AvailableCrds[VolumeGroupSnapshotClassCrdName] {
 			csiOperatorConfig.Spec.DriverSpecDefaults.SnapshotPolicy = csiopv1a1.VolumeGroupSnapshotPolicy
 		}
+		if val, ok := c.operatorConfigMap.Data[csiCtrlPluginHostNetworkKey]; ok && val == strconv.FormatBool(false) {
```

Comment:
- There is no point in checking the existence of the key if we only care about a true value as string 
- Please use a case insensitive compare operation, preferably convert the key into a bool and not the other way around: 
```suggestion
		useHostNetworkForCsiControllers := strconv.ParseBool(c.operatorConfigMap.Data[csiCtrlPluginHostNetworkKey])
		if !useHostNetworkForCsiControllers {
```